packages/openapi-ts-tests/main/test/__snapshots__/2.0.x/plugins/@pinia/colada/asClass/@pinia/colada.gen.ts

```diff
@@ -41,19 +41,13 @@ const createQueryKey = <TOptions extends Options>(id: string, options?: TOptions
 
 export const getFooQueryKey = (options?: Options<GetFooData>) => createQueryKey('getFoo', options);
 
-export const getFooQuery = (options?: Options<GetFooData>): UseQueryOptions<GetFooResponse, Error> => {
-    return {
-        key: getFooQueryKey(options),
-        query: async (context) => {
-            const { data } = await FooBazService.getFoo({
-                ...options,
-                ...context,
-                throwOnError: true
-            });
-            return data;
-        }
-    };
-};
+export const getFooQuery = defineQueryOptions({
```

Comment:
Is there an equivalent of `defineQueryOptions` for mutations?

Reply:
I think you summarized it pretty well. It's all about typed cache access with `querycache.getData(definedOptions(id).key)`.